src/ui/components/Comments/useUserCommentPreferences.ts

```diff
@@ -0,0 +1,24 @@
+import { useMemo } from "react";
+
+import useLocalStorage from "replay-next/src/hooks/useLocalStorage";
+
+type Filter = "current-user" | null;
+type SortBy = "created-at" | "recording-time";
+
+export default function useUserCommentPreferences() {
+  const [filter, setFilter] = useLocalStorage<Filter>("Replay:CommentPreferences:Filter", null);
+  const [sortBy, setSortBy] = useLocalStorage<SortBy>(
+    "Replay:CommentPreferences:sortBy",
+    "recording-time"
+  );
```

Comment:
Here's a question: Is this a _per recording_ preference or a global preference? My initial inclination was global, but I'm not 100% sure.

Reply:
Hmm. My own instinct says it makes more sense per-recording, but I'd be okay doing it globally as a starting point.